### PR TITLE
Added issue templates to report problems and ask for new features.

### PR DESCRIPTION
**Pull request by raquel-ucl**
Helps with #45 and #56

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/I_need_help.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name: I need help
+about:  Write to Nisaba developers about a problem you have encountered
+
+---
+
+**Describe the problem**
+A clear and concise description of what the problem is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (ageorgou) [inline .github/ISSUE_TEMPLATE/I_need_help.md:11]: Maybe add "e.g." or something to make it clear they don't have to be exactly these steps?

File: .github/ISSUE_TEMPLATE/I_need_help.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (ageorgou) [inline .github/ISSUE_TEMPLATE/I_need_help.md:21]:
```suggestion
If applicable, add screenshots to help explain your problem (follow these [instructions](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files) if you need help).
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (ageorgou) [inline .github/ISSUE_TEMPLATE/feature_request.md:8]:
```suggestion
A clear and concise description of what the problem is. For example, "I'm always frustrated when [...]"
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (ageorgou) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: Maybe this is too repetitive or may put people off?

**Review by ageorgou — APPROVED** [review]
The only comment I have is that "clear and concise", especially repeated, comes across a bit aggressive, and could discourage people from reporting problems/suggestions. But I may be overthinking it...

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 4 above)
Comment (raquel-ucl) [reply .github/ISSUE_TEMPLATE/feature_request.md:14]: You're right! Rewriting...

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+Please describe the problem is. For example, "I'm always frustrated when [...]"
 
 **Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+How would it look like? How would a user interact with this feature? What is its expected behaviour?
 
 **Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+Please describe alternative solutions or features you've considered and why you would preferred the solution described above.
```
Comment (ageorgou) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]:
```suggestion
Please describe alternative solutions or features you've considered and why you would prefer the solution described above.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
 
 **Additional context**
-Add any other context or screenshots about the feature request here.
+Add any other context, screenshots, schemas, etc. about the feature request here. You can follow these [instructions](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files) in order to upload files here.
```
Comment (ageorgou) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]:
```suggestion
Add any other context, screenshots, diagrams, etc. about the feature request here. You can follow these [instructions](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files) in order to upload files here.
```

**Review by ageorgou — COMMENTED** [review]
Just small suggestions!

**Pull request merged**